packages/eslint-plugin/src/stylex-margin-padding-shorthand.js

```diff
@@ -72,143 +80,161 @@ const shorthandAliases = {
 };
 
 const stylexMarginPaddingShorthand = {
-    meta: {
-      messages: {
-        noMarginShorthands:
-          'Property shorthands like `{{property}}: 0 0 0 0` are not supported in styleX. Separate into `{{property}}InlineStart`, `{{property}}InlineEnd` [...].',
-        noLegacyFormats:
-          'Use {{correctProperty}} instead of legacy formats like {{incorrectProperty}} to adhere to logical property naming.',
-      },
-      fixable: 'code',
+  meta: {
+    messages: {
+      noMarginShorthands:
+        'Property shorthands like `{{property}}: 0 0 0 0` are not supported in styleX. Separate into individual properties.',
+      noLegacyFormats:
+        'Use {{correctProperty}} instead of legacy formats like {{incorrectProperty}} to adhere to logical property naming.',
```

Comment:
```suggestion
        `Property shorthands using multiple values like "${property}: 0 0 0 0" are not supported in StyleX. Separate into individual properties.',
      noLegacyFormats:
        `Use "${correctProperty}" instead of legacy formats like "${incorrectProperty}" to adhere to logical property naming.',
```

Reply:
these are actually unused from me porting over this lint rule from www (will remove!), where `messageId` syntax to context.report is supported due to a newer version of eslint being used there, although {{}} is the expected format to create a message template